### ISSUE TITLE
[sub-issue] CRw: Color contrast in tab navigation is insufficient

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#281]
![image](https://cloud.githubusercontent.com/assets/7510842/5357053/5c20cd20-7fa3-11e4-8d78-9a6961a8adc8.png)

The tab navigations have insufficient color contrast.
http://swiss.github.io/styleguide/navigation_modules_-_content_navigation.html#d-tab-nav
http://swiss.github.io/styleguide/content_modules_-_content_containers.html#b-further-info

==> Use #454545 for the Header texts in Tabs

**Comment by tonifisler** [comment]
This is actually a duplicate of #324. Closing.
